x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1633,13 +1632,6 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
 
     await this.update(soClient, esClient, id, updatePackagePolicy, updateOptions);
 
-    // Persist any experimental feature opt-ins that come through the upgrade process to the Installation SO
-    await updateDatastreamExperimentalFeatures(
```

Comment:
There is no reason for this to change during an upgrade and that codepath was extremely slow

Reply:
We do not, but it seems that codepath was doing nothing, it seems it was only here for supporting an UI that do not exists anymore https://github.com/elastic/kibana/issues/190613

Reply:
Thanks for checking!